backend/config.json.example

```diff
@@ -118,7 +118,8 @@
       "type": "vault",
       "host": "http://vault1:8200/",
       "prefix": "/v1/",
-      "backend": "secret/"
+      "backend": "secret/",
+      "ttl": 600000
```

Comment:
Do you think it would be a good idea to turn this into a ´.js` file to add some comments? For example for this field i'd like a little comment along the lines of `time in ms until vault's refresh endpoint is called`.

Also, would it might be better to specify the ttl in seconds or minutes - what do you think?